### PR TITLE
Motion component

### PR DESCRIPTION
**Pull request by ryan-hunt-priceline**

### CODE REVIEW

**Comment by craigpalermo** [comment]
- [ ] `rush change`

**Review by craigpalermo — APPROVED** [review]
LGTM once @wesleymartin85 approves the new stories in Chromatic.

**Review by wesleymartin85 — APPROVED** [review]

**Pull request merged**
